src/test/java/core/basesyntax/service/impl/CsvReportServiceImplTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReportService;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvReportServiceImplTest extends CsvReportServiceImpl {
+    private ReportService csvReportService;
+    private String expected;
+    private Map<String, Integer> map;
+
+    @Before
+    public void setUp() {
+        csvReportService = new CsvReportServiceImpl();
+        map = new HashMap<>();
+        map.put("banana", 40);
+        map.put("apple", 20);
+        expected = "fruit,quantity" + System.lineSeparator() + "banana,40"
+                + System.lineSeparator() + "apple,20";

```

Comment:
```suggestion
    private String expected = "fruit,quantity" + System.lineSeparator() + "banana,40"
                + System.lineSeparator() + "apple,20";
    private Map<String, Integer> map = Map.of(
            "banana", 40,
            "apple", 20
    );
    
    @Before
    public void setUp() {
        csvReportService = new CsvReportServiceImpl();
```
Just a suggestion (you can live it like you did it)
collections, string values, numbers, objects like Random can be initialized in fields,
different objects of services, dao ..., have to be initialized in constructors or methods like this (annotated with Before or BeforeClass)